src/rules/no-reactive-literals.ts

```diff
@@ -0,0 +1,69 @@
+import type { TSESTree } from "@typescript-eslint/types"
+import { createRule } from "../utils"
+
+const labeledStatementBase = `SvelteReactiveStatement > ExpressionStatement > AssignmentExpression`
+
+export default createRule("no-reactive-literals", {
+  meta: {
+    docs: {
+      description: "Don't assign literal values in reactive statements",
+      category: "Stylistic Issues",
```

Comment:
Probably not a stylistic issue. I think this rule should be `"Best Practices"`.

```suggestion
      category: "Best Practices",
```